packages/www/pages/dashboard/developers/webhooks/[id].tsx

```diff
@@ -247,21 +247,23 @@ const WebhookDetail = () => {
                           STATUS_CODES[data.status.lastFailure.statusCode]
                         }`}</Cell>
                     </>
-                  ) : (
+                  ) : data.status.lastFailure ? (
                     <>
                       <Cell variant="gray">Error message</Cell>
                       <Cell
                         css={{
                           fontFamily: "monospace",
                         }}>
-                        {data.status.lastFailure.error}
+                        {data.status.lastFailure?.error}
```

Comment:
`lastFailure` can't be null/undefined here since you just checked above. We might want to have a fallback tho in case the error itself is null wdyt?
```suggestion
                        {data.status.lastFailure.error ?? "unknown"}
```

Reply:
Added!